packages/core-ui/lib/index.tsx

```diff
@@ -48,18 +48,21 @@ export { ModalSeparator } from "./modal/ModalSeparator";
 export { ModalWarnings } from "./modal/ModalWarnings";
 
 // Buttons
-export { ButtonPrimary } from "./buttons/ButtonPrimary";
-export { ButtonPrimaryInvert } from "./buttons/ButtonPrimaryInvert";
-export { ButtonSecondary } from "./buttons/ButtonSecondary";
-export { ButtonTertiary } from "./buttons/ButtonTertiary";
-export { ButtonMono } from "./buttons/ButtonMono";
-export { ButtonMonoInvert } from "./buttons/ButtonMonoInvert";
-export { ButtonSubmitInvert } from "./buttons/ButtonSubmitInvert";
-export { ButtonText } from "./buttons/ButtonText";
-export { ButtonSubmit } from "./buttons/ButtonSubmit";
-export { ButtonHelp } from "./buttons/ButtonHelp";
-export { ButtonTab } from "./buttons/ButtonTab";
-export { ButtonOption } from "./buttons/ButtonOption";
+export { Button } from "./buttons";
+
+// TRIAGE: I do not think that there is a need for exporting the buttons below - maybe for backward compatibility
+// export { ButtonPrimary } from "./buttons/ButtonPrimary";
+// export { ButtonPrimaryInvert } from "./buttons/ButtonPrimaryInvert";
+// export { ButtonSecondary } from "./buttons/ButtonSecondary";
+// export { ButtonTertiary } from "./buttons/ButtonTertiary";
+// export { ButtonMono } from "./buttons/ButtonMono";
+// export { ButtonMonoInvert } from "./buttons/ButtonMonoInvert";
+// export { ButtonSubmitInvert } from "./buttons/ButtonSubmitInvert";
+// export { ButtonText } from "./buttons/ButtonText";
+// export { ButtonSubmit } from "./buttons/ButtonSubmit";
+// export { ButtonHelp } from "./buttons/ButtonHelp";
+// export { ButtonTab } from "./buttons/ButtonTab";
+// export { ButtonOption } from "./buttons/ButtonOption";
```

Comment:
I would still like to use these exports in the staking dashboard, I would like to think the standalone `Button` component as an optional utility. XCM dashboard, absolutely let's use it there, but would like things to remain the same in staking for the timebeing.

Reply:
That was my understanding, thus I added the "triage" title. Will uncomment, and once this is released, we could alter the Dashboard as well following the above pattern